src/olympia/addons/cron.py

```diff
@@ -226,9 +227,9 @@ def unhide_disabled_files():
     files = set(File.objects.filter(q | Q(status=amo.STATUS_DISABLED))
                 .values_list('version__addon', 'filename'))
     for filepath in walkfiles(settings.GUARDED_ADDONS_PATH):
-        addon, filename = filepath.split('/')[-2:]
+        addon, filename = force_text(filepath).split('/')[-2:]
         if tuple([int(addon), filename]) not in files:
-            log.warning(u'File that should not be guarded: %s.', filepath)
+            log.warning('File that should not be guarded: %s.', filepath)
```

Comment:
I'd prefer to `force_text` on `filepath` at the beginning of the loop and keep the `u` prefix there and below.